### PR TITLE
Berlin WDFT - Jan Colomer

### PR DESCRIPTION
**Pull request by Jancololo**

### CODE REVIEW

File: src/movies.js [hunk 1]
```diff
@@ -1,16 +1,147 @@
...
+//         }
+//     })
+
+//     const firstTwentyMoviesTitle = firstTwentyMovies.map(function (movie) {
+//         return movie.title
+//     })
+//     return firstTwentyMoviesTitle
+// }
+
+function orderAlphabetically(moviesArr) {
```
Comment (annawaszak) [inline src/movies.js:99]: You passed "moviesArr" as a parameter, but inside the function you're mapping over "movies".

File: src/movies.js [hunk 2]
```diff
@@ -1,16 +1,147 @@
...
+    }).sort(function (a, b) {
+        return a.localeCompare(b);
+    }).slice(0, 20);
+
+    return ordered;
+}
+
 // BONUS - Iteration 7: Time Format - Turn duration of the movies from hours to minutes
 
+function turnHoursToMinutes(movies) {
```
Comment (annawaszak) [inline src/movies.js:111]: Great work!

File: src/movies.js [hunk 3]
```diff
@@ -1,16 +1,147 @@
...
+    });
+
+    const best = averageRatingsPerYear.sort(function (a, b) {
+        if (a.rate === b.rate) {
+            return a.year - b.year
+        }
+        return b.rate - a.rate;
+    })[0];
+    return `The best year was ${best.year} with an average rate of ${best.rate}`;
+}
```
Comment (annawaszak) [inline src/movies.js:147]: Awesome job! This was a complicated one and you made it seem easy.

**Review by annawaszak — COMMENTED** [review]
Hi Jan,

You did a fantastic job on this lab! You can find my comments below (mostly flattering as it was hard to find anything to improve). 
As always, I remain at your disposal if you had any questions/suggestions/feedback.

Thanks,
Anna

File: src/movies.js (hunk 3 above)
Comment (Jancololo) [reply src/movies.js:147]: Thanks for the feedback!

**Pull request closed without merging**